fips_cli.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
+from typing import Optional, List, Set
+import click
+from pathlib import Path
+import logging
+import sys
+import os
+from datetime import datetime
+
+from sec_certs.configuration import config
+from sec_certs.dataset.fips import FIPSDataset
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument(
```

Comment:
Or maybe it should be because you can have multiple actions on the CLI as arguments. Anyway, have a look at Click whether there is anything that can help this CLI implementation.